deploy/common/k8sbased.sh

```diff
@@ -55,6 +55,12 @@ function hotload_binary {
   echo
 
   binary_path=$(realpath "$(git rev-parse --show-toplevel)/bin/linux_amd64/${local_name}")
+  # set custom source path, e.g. when StackRox source is mounted into kind.
+  if [[ -n "$ROX_LOCAL_SOURCE_PATH" ]]; then
```

Comment:
Hmm, I personally would have used something like `${ROX_LOCAL_SOURCE_PATH:-}` for variables that are not properly initialized anywhere (to an empty string for example). Code like this would fail under `set -u`, but, it seems that there are different opinions on this matter, so I just leave this one as an FYI here.